README.md

```diff
@@ -51,6 +53,15 @@ You can also run the tool with optional arguments:
 ./deposit.sh --num_validators=<NUM_VALIDATORS> --mnemonic_language=english --password=<YOUR_PASSWORD> --folder=<YOUR_FOLDER_PATH>
 ```
 
+#### Arguments
+
+| Argument | Optional | Description |
+| -------- | -------- | -------- |
+| `--num_validators`  | Y  | The number of signing keys you want to generate. Note that the child key(s) are generated via the same master key. |
+| `--mnemonic_language` | N, default to `english` | The mnemonic language. Options: `czech`, `chinese_traditional`, `chinese_simplified`, `english`, `spanish`, `italian`, `korean` |
```

Comment:
```suggestion
| `--mnemonic_language` | Y, default to `english` | The mnemonic language. Options: `czech`, `chinese_traditional`, `chinese_simplified`, `english`, `spanish`, `italian`, `korean` |
```

Reply:
Maybe it is worth distinguishing between Optional when calling the CLI and entirely optional (as if if you don't say anything, the CLI makes an assumption on your behalf).